frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/Toolbar/MobileToolbar.tsx

```diff
@@ -23,6 +23,15 @@ export const MobileToolbar: FC = () => {
     setHasInteracted(true)
   }
 
+  useEffect(() => {
+    const handleResize = () => {
+      setHasInteracted(false)
+    }
+
+    window.addEventListener('resize', handleResize)
+    return () => window.removeEventListener('resize', handleResize)
+  }, [])
```

Comment:
this is for `2. change responsiveness` . How do you think?

Reply:
Looks good!!